src/display/editor/editor.js

```diff
@@ -1049,6 +1049,14 @@ class AnnotationEditor {
     this.#altText?.finish();
   }
 
+  /**
+   * Get the toolbar buttons for this editors.
+   * @returns {Array<Array<string|object>>|null}
+   */
+  get toolbarTools() {
```

Comment:
Should we perhaps call this `get toolbarButtons()` since that seems to match the docstring and usage below more closely? It would make the `const toolbarButtons = this.toolbarTools;` line below a bit more intuitive if it reads like `const toolbarButtons = this.toolbarButtons;`.